components/library/exiv2/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Carsten Grzemba
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= exiv2
+COMPONENT_VERSION= 0.25
+COMPONENT_SUMMARY= EXIF tag parsing library for digital cameras
+COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH= \
+  sha256:c80bfc778a15fdb06f71265db2c3d49d8493c382e516cb99b8c9f9cbde36efa4
+COMPONENT_ARCHIVE_URL= \
+  http://www.exiv2.org/$(COMPONENT_ARCHIVE)
+
+COMPONENT_PROJECT_URL = http://www.exiv2.org
+COMPONENT_FMRI = image/library/exiv2
+COMPONENT_LICENSE = GPLv2
+COMPONENT_LICENSE_FILE = COPYING
+COMPONENT_CLASSIFICATION = System/Multimedia Libraries
+
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/configure.mk
+include $(WS_TOP)/make-rules/ips.mk
+
+COMPONENT_PRE_CONFIGURE_ACTION = \
+     $(CLONEY) $(SOURCE_DIR) $(@D)
+
+# linking 64bit '-m64' option is missing
+LDFLAGS += -m$(BITS) -lnsl -lsocket 
+
+# using mkdir option '-v'
+COMPONENT_BUILD_ARGS += PATH="/usr/gnu/bin:$(PATH)"
```

Comment:
Perhaps, better do it on top level? Just set 
PATH=$(PATH.gnu)
in the beginning (somewhere around 37 line).